docs/src/go-client/get-started.md

```diff
@@ -17,34 +17,16 @@ You will be guided through the following steps:
 
 ## Prerequisites
 
-* Go v1.9+ environment installed
+* Go v1.11+ environment installed
 * [Zeebe distribution](../introduction/install.html)
 * [Zeebe Modeler](https://github.com/zeebe-io/zeebe-modeler/releases)
 * [Zeebe Monitor](https://github.com/zeebe-io/zeebe-simple-monitor/releases)
 
 
-Now, start the Zeebe broker.
-
-In case you want to create another topic you can use `zbctl` from the `bin` folder.
-Create the topic with zbctl by executing the following command on the command line:
-
-```
-zbctl create topic my-topic --partitions 1
-```
-
-You should see the output:
-
-```
-{
-  "Name": "my-topic",
-  "Partitions": 1,
-  "ReplicationFactor": 1
-}
-```
-
-**Note:** You can find the `zbctl` binary in the `bin/` folder of the Zeebe
-distribution. On Windows systems the executable is called `zbctl.exe` and on
-MacOS `zbctl.darwin`.
+Before you begin to setup your project please start the broker, i.e. by running the start up script 
+`bin/broker` or `bin/broker.bat` in the distribution. Per default the broker is binding to the 
+address `localhost:26500`, which is used as contact point in this guide. In case your broker is 
```

Comment:
Not 26501?

Reply:
The gateway port is 26500. The broker address is 26501 but the clients are not connecting to that port.

Reply:
Yeah I would like to have that :+1:

Reply:
https://github.com/zeebe-io/zeebe/issues/1516